dbt/adapters/databricks/impl.py

```diff
@@ -293,24 +303,32 @@ def execute(
 
     def list_relations_without_caching(  # type: ignore[override]
         self, schema_relation: DatabricksRelation
-    ) -> list[DatabricksRelation]:
-        empty: list[tuple[Optional[str], Optional[str], Optional[str], Optional[str]]] = []
+    ) -> List[DatabricksRelation]:
```

Comment:
You are not importing any `List` object from `typing`.


```suggestion
    ) -> list[DatabricksRelation]:
```

This should still be fine, because Python 3.11 supports the typing info on the objects themselves.